plugins/formal-verification/src/org/jetbrains/kotlin/formver/conversion/MethodConversionContext.kt

```diff
@@ -16,18 +18,27 @@ import viper.silver.ast.Method
  * only need the signature in most methods, as we verify methods one at a
  * time.
  */
-class MethodConversionContext(val programCtx: ProgramConversionContext, val signature: ConvertedMethodSignature, val body: FirBlock?) {
+class MethodConversionContext(
+    val programCtx: ProgramConversionContext,
+    val signature: ConvertedMethodSignature,
+    val body: FirBlock?,
+    val effects: List<FirEffectDeclaration> = listOf()
```

Comment:
Probably makes more sense to take a `FirResolvedContractDescription` here and unpack it inside.